core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SampleStockSingleCell.java

```diff
@@ -0,0 +1,21 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
+
+import java.math.BigDecimal;
+
+public interface SampleStockSingleCell extends SampleStock {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)